Updates to onboarding for Amazon Sales Channel (#377)
magento/merchdocs: src/sales-channels/asc/ob-create-listing-rule.md

---
title: 'Onboarding: Create Listing Rule'
---

Listing rules are initially defined during onboarding, but can be modified at any time after your store setup is complete. After onboarding, you can access the listing rules for an Active or Inactive store in the store’s dashboard. For more information, see [Listing Rules]({% link sales-channels/asc/listing-rules.md %}).

## Create a listing rule during onboarding

1. Click <span class="btn">View Store</span> for the added store.

    The store's dashboard appears with the `No products listed to Amazon` message.

1. Click <span class="btn">Preview and List Eligible Products</span>.

    The Listing Rules page appears.

> 1. Define your desired conditions for the eligibility of products to be listed on Amazon and click <span class="btn">Preview changes</span>. You can also click <span class="btn">Preview changes</span> to skip this step.

Reviewer: ```suggestion
1. Define your desired conditions for the eligibility of products to be listed on Amazon and click <span class="btn">Preview changes</span>, or click <span class="btn">Preview changes</span> to skip this step.
```
